coalahtml/helper.py

```diff
@@ -95,4 +133,11 @@ def get_args():
                             metavar='BOOL',
                             default=False,
                             help='Do not launch webserver.')
+    arg_parser.add_argument('--noreload',
+                            nargs='?',
+                            const=True,
+                            metavar='BOOL',
+                            default=False,
+                            help='Do not update the local copy of app with'
+                                 'latest changes.')
```

Comment:
Also what about mentioning when it can be useful here? Not a lot, maybe some small sentence.